packages/legend-graph/src/helpers/DomainHelper.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Class } from '../models/metamodels/pure/packageableElements/domain/Class';
+
+export const isVersionedClass = (val: Class): boolean => {
```

Comment:
Let's add a unit test for this. Very easy to test with test data -> building graph and a direct call on the function.